Assets/Scripts/Models/Furniture.cs

```diff
@@ -1105,7 +1104,7 @@ private bool DefaultIsValidPosition(Tile tile)
                 }
 
                 // Make sure tile is FLOOR
-                if (t2.Type != TileType.Floor && tileTypeBuildPermissions.Contains(t2.Type) == false)
+                if (t2.Type != TileType.Floor && tileTypeBuildPermissions.Contains(t2.Type.Type) == false)
```

Comment:
It should work without this.
`TileType` already knows how to compere it with `TileType`:
```
public bool Equals(TileType other)
{
    return string.Equals(Type, other.Type);
}

public override bool Equals(object obj)
{
    return Equals((TileType)obj);
}
```